src/view/com/post-thread/PostThread.tsx

```diff
@@ -367,7 +367,7 @@ export const PostThread = observer(function PostThread({
       data={posts}
       initialNumToRender={posts.length}
       maintainVisibleContentPosition={
-        isNative && view.isFromCache
+        isNative && view.isFromCache && view.isCachedPostAReply
```

Comment:
Wanna leave a comment mentioning the other case is broken? So that it's clear why the condition

Reply:
The other case isn't broken in the sense that this is a hackfix. This change is just another way to do it correctly -- if the post we're loading isnt a reply, we don't want to reposition.